.github/workflows/tests.yml

```diff
@@ -11,6 +11,7 @@ on:
 env:
   CARGO_INCREMENTAL: 0
   RUSTFLAGS: "-Dwarnings"
+  EMSDK_VERSION: 1.39.20 # Last emsdk compatible with Rust's LLVM 11
```

Comment:
I think it would be better to move this variable to the `emscripten-tests` job, since it's the only affected job.

Reply:
I forgot jobs can have `env`s, good catch